libs/GasBurnerControl/GasBurnerControl.h

```diff
@@ -0,0 +1,69 @@
+#ifndef GasBurnerControl_h
+#define GasBurnerControl_h
+
+
+#include "Arduino.h"
```

Comment:
```suggestion
#include <Arduino.h>
```

Reply:
Done in 87de580

I have also removed the duplicate include in GasBurnerControl.cpp